src/site/apt/index.apt.vm

```diff
@@ -26,14 +26,14 @@
 ${project.name}
 
   The Toolchains Plugins enables sharing configuration across plugins,
-  for example to make sure the plugins like compiler, surefire, javadoc, webstart etc. all use the same JDK for execution.
+  for example to make sure the plugins like compiler, surefire, javadoc, webstart etc. all use the same JDK for execution independantly from JRE used to run Maven itself.
   Similarly to the maven-enforcer-plugin, it allows you to control environmental constraints in the build.
 
 * Goals Overview
 
   The Toolchains plugin has one goal:
 
-  * {{{./toolchain-mojo.html}toolchains:toolchain}} checks that toolchains requirements are met by currently configured toolchains.
+  * {{{./toolchain-mojo.html}toolchains:toolchain}} selects a toolchain based on configured requirements and store it in build context for later retrieval by other plugins.
```

Comment:
If [it] `select*s*` then it `store*s` I guess.

Reply:
fixed, thank you